worker/include/RTC/Codecs/PayloadDescriptorHandler.hpp

```diff
@@ -14,6 +16,65 @@ namespace RTC
 			virtual void Dump() const    = 0;
 		};
 
+		class PictureIdList
+		{
+			static constexpr uint16_t MaxCurrentLayerPictureIdNum{ 1000u };
```

Comment:
@vpalmisano I think a much lower value here is possible: ie 64 or 128. It'ts 64 or 128 changes or layers...

Reply:
But here we use that value to remove older pictureId values, it doesn't represent the number of layer changes.

Reply:
yep, but we just add new pictureId values when changing layers, don't we?

Reply:
Yes but MaxCurrentLayerPictureIdNum is used to evaluate the distance in packets between the last and the first item in the list.

Reply:
The value `MaxCurrentLayerPictureIdNum` merely indicates how long the `std::deque` can be. We can make it smaller. I think 128 would be a good value, even 64, as older picture IDs will anyway be too old to consider.

Reply:
But why a 'counter'? My only concern is, can we make `MaxCurrentLayerPictureIdNum` smaller than the current `1000`? I mean just decrease that value.

Reply:
It we have a layer change every 1000 packets, we will hold only 1 item in the queue and it would be not useful.

Reply:
Current `MaxCurrentLayerPictureIdNum` is 1000. If we have a layer change every 1000 packets we will keep pictureId <-> layer relation for the last  `MaxCurrentLayerPictureIdNum` (1000) changes. This is, we will know if we must relay or drop the last 1M packets.

If we set `MaxCurrentLayerPictureIdNum` to 64 and we have a layer change every 1000 packets we will keep pictureId <-> layer relation for the last `MaxCurrentLayerPictureIdNum` (64) changes. This is, we will know if we must relay or drop the last  64000 packets.

That is correct, right?

If so, I consider that being able to relay or drop the last 1M packets is not needed, and a lower value of `MaxCurrentLayerPictureIdNum` would suffice.

Reply:
Why?
Every time we have a layer change at `pictureId` we put a new item in the queue and we remove the last one if the difference is larger than 1000 packets:

```
					uint16_t diff = pictureId - it.first
					if (diff > MaxCurrentLayerPictureIdNum)
					{
						this->list.pop_front();
					}
```
With a layer change every 1000 packets we will keep only 1 item in the queue. Right?

Reply:
> With a layer change every 1000 packets we will keep only 1 item in the queue. Right?

Correct, yes. So unless there is a layer change for every new pictureID, we will never get 1000 items in the queue...

A value of 1000 here doesn't seem that big as it does not mean there will be those many items in the queue. 

👌